test/internet/test-single-executable-application.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
```

Comment:
Does anyone know how I can run this test on CI? I have placed it inside `test/internet` instead of `test/parallel` because it attempts to do `npx postject` which makes an outbound network connection.

Reply:
I'm thinking VCR_Cassettes, like in Ruby?

Reply:
On Node's CI runners? Or any sort of CI?

Reply:
During the single-executable meeting, we were discussing about including postject inside `test/fixtures`, just for testing. So that might solve this problem.